ruma-events/benches/event_deserialize.rs

```diff
@@ -0,0 +1,130 @@
+// In order to run use either `cargo bench` or to save if you
+// want to save the result to compare branches use
+// `cargo bench --bench event_deserialize -- --save-baseline test-output`
+// Since we are in a workspace the default `cargo bench` still picks up
+// the args passed to it to avoid this use the above command.
+
+use criterion::{criterion_group, criterion_main, Criterion};
+use ruma_events::{
+    room::{message::MessageEventContent, power_levels::PowerLevelsEventContent},
+    AnyRoomEventStub, EventJson, MessageEvent, StateEventStub,
+};
+use serde_json::json;
+
+fn deserialize_any_event_stub(c: &mut Criterion) {
```

Comment:
Could we test all of `AnyEvent`, `AnyRoomEvent` and `AnyStateEvent`? Then we'd have all levels of specificity and could see how the amount of hard-coded knowledge about what we're deserializing affects performance.

```suggestion
fn deserialize_any_event_stub(c: &mut Criterion) {
    ...
}

fn deserialize_any_room_event_stub(c: &mut Criterion) {
```